src/Sulu/Bundle/AudienceTargetingBundle/DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AudienceTargetingBundle\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+/**
+ * Configuration definition of sulu_audience_targeting.
+ */
+class Configuration implements ConfigurationInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getConfigTreeBuilder()
+    {
+        $treeBuilder = new TreeBuilder();
+        $rootNode = $treeBuilder->root('sulu_audience_targeting');
+
+        $rootNode
+            ->children()
+                ->scalarNode('number_of_priorities')
+                    ->defaultValue(5)
+                ->end()
+            ->end();
+
+        $this->addObjectsSection($rootNode);
+
+        return $treeBuilder;
+    }
+
+    /**
+     * Adds `objects` section.
+     *
+     * @param ArrayNodeDefinition $node
+     */
+    private function addObjectsSection(ArrayNodeDefinition $node)
+    {
+        $node
+            ->children()
+                ->arrayNode('objects')
+                    ->addDefaultsIfNotSet()
+                    ->children()
+                        ->arrayNode('target_group')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->scalarNode('model')
+                                    ->defaultValue('Sulu\Bundle\AudienceTargetingBundle\Entity\TargetGroup')
```

Comment:
Couldn't we also use the `TargetGroup::class` variant in all appearances of this file?